tools/upload_file.php

```diff
@@ -0,0 +1,122 @@
+<?php
```

Comment:
If this is only for resumable files, calling it `upload_resumable_file.php` would be more accurate and informative.